src/app/pages/resources/overview/overview.jsx

```diff
@@ -28,22 +32,43 @@ const useStyles = makeStyles({
   pageBody: {
     display: 'grid',
     gap: '16px',
-    gridTemplateColumns: '1fr 4fr',
+    gridTemplateColumns: '1fr',
     padding: '32px',
     backgroundColor: '#f9f9f9',
+    [theme.breakpoints.up('md')]: {
+      gridTemplateColumns: '1fr 4fr',
+    },
   },
   content: {
     display: 'grid',
-    gap: '32px',
+    gap: '8px',
```

Comment:
Why is this a grid at all? It's just a block of Typography blocks. Does it need this styling?

Reply:
The grid was used to create spacing between typography elements as you can see im using gap to create the spacing.